### DISCUSSION TITLE
Evapotranspiration in StressModels?

### DISCUSSION THREAD

**Discussion by Sarali97** [pastas/pastas#300] [Q&A] [answered]
Hello..
I am currently encountrering a problem regarding the StressModels. Depending on which one I chose I obtain different answers. According to the documentation you should use Reference EvapoTranspiration (RET) when using the RechargeModel (RM) and Potential Evaporation(PE) using StressModel (SM) and StressModel2 (SM2). 

However, when using RET I obtain the same results for SM2 and RM, which makes me think that SM2 also uses RET which is not stated in the doc. This makes me suspect that SM also uses it (however, when using RET for SM I get different results than from the previous stressmodels)

My question is; what do you mean by "Potential Evaporation" which is stated throughout the documentation, does it mean PET or RET or is it just PE. I am very confused. :(

Best regards, 
S

**Comment by mbakker7** [comment, marked as answer]
Sarali,

Thanks for your report.
I am not sure what the exact diff is between potential evap and reference
evap. I assume their time series are very similar. So they should give
similar results. Are they very different for you?

The docs should probably be consistent in using the term reference or
potential evap.
